roles/openshift_storage_nfs/defaults/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+exports_dir: /var/export
+registry_volume: regvol
+export_options: '*(rw,sync,all_squash)'
+os_firewall_use_firewalld: False
+os_firewall_allow:
+- service: portmapper
+  port: "111/tcp"
+- service: nfs
+  port: "2049/tcp"
+- service: mount lock daemon
+  port: "20048/tcp"
+- service: status
+  port: "50825/tcp"
+- service: frsrpc
+  port: "53248/tcp"
```

Comment:
I was using @thoraxe's guide for most of this 8)

https://github.com/openshift/training/blob/master/07-Registry-and-Storage.md